client/src/partials/accounts/create_account/create.js

```diff
@@ -135,6 +144,10 @@ angular.module('bhima.controllers')
       session.accountClass = $scope.checkClass($scope.newAccount.number);
     }
 
+    function typeAccountClass () {
```

Comment:
This function only does one operation.  Why do you need to create a new function for it?